app/components/article-wrapper.js

```diff
@@ -86,6 +86,10 @@ export default Component.extend(
 			}
 		}),
 
+		featuredVideoVisible: computed('model.hasPortableInfobox', 'model.featuredVideo', function () {
```

Comment:
What do you think about:
noPortableInfobox: computed.not('model.hasPortableInfobox'),
featuredVideoVisible: computed.and('noPortableInfobox', 'model.featuredVideo')